istio-control/istio-config/templates/poddisruptionbudget.yaml

```diff
@@ -15,6 +15,6 @@ spec:
       app: galley
       release: {{ .Release.Name }}
       istio: galley
+      version: {{ .Values.version }}
```

Comment:
`version` has no default, will this break? Because I think this will template to `version: ` and `name: istio-galley-`.

Reply:
I added version ('default') to global.yaml.

I'm not entire sure what's the best option - I can add 'if' and some extra to add the '-' where needed, to preserve the old behavior ( of not having a version ).